testsuite/cluster-test/src/instance.rs

```diff
@@ -271,6 +272,42 @@ impl Instance {
             )
             .await
     }
+
+    /// Unlike util_cmd, exec runs command inside the container
+    pub async fn exec(&self, command: &str) -> Result<()> {
+        let child = Command::new("kubectl")
+            .arg("exec")
+            .arg(&self.peer_name)
+            .arg("--container")
+            .arg("main")
```

Comment:
Can you update the LSR container to be named `main` as well?

Reply:
done